platform/security/sts/security-sts-x509validator/src/main/java/org/codice/ddf/security/validator/x509/X509PathTokenValidator.java

```diff
@@ -271,9 +258,30 @@ public TokenValidatorResponse validateToken(TokenValidatorParameters tokenParame
             }
 
             Credential returnedCredential = validator.validate(credential, requestData);
+            X500Principal subjectX500Principal =
+                    returnedCredential.getCertificates()[0].getSubjectX500Principal();
             response.setPrincipal(returnedCredential.getCertificates()[0].getSubjectX500Principal());
```

Comment:
Can't we just use the subjectX500Principal here and below instead of pulling it out of the returnedCredentials every time?